packages/shylu/shylu_dd/bddc/src/shylu_PreconditionerBDDC.h

```diff
@@ -68,15 +80,16 @@ using Teuchos::rcp;
 
 // Author: Clark R. Dohrmann
 namespace bddc {
-
-template <class SX, class SM, class LO, class GO>
-  class PreconditionerBDDC
+  
+template <class SX, class SM, class LO, class GO> 
+  class PreconditionerBDDC : 
+  public PreconditionerBase<SX,SM,LO,GO>
 {
 public:
   //
   // Convenience typedefs
   //
-  typedef Tpetra::Map<>::node_type  Node;
+  typedef Tpetra::Map<>::node_type                                Node;
   typedef Tpetra::Map<LO,GO,Node>                                 Map;
```

Comment:
See comment above.  If you only ever use the default Node type, then you should not specify the Node template parameter.

Reply:
I think I removed all references to Node in all the files as you suggested.